src/sonic_exporter/config/logging.yml

```diff
@@ -1,13 +1,6 @@
 version: 1
 
 handlers:
-    file:
```

Comment:
@Cellebyte File based logs are better for debugging, docker logs or consol logs and quickly grow big also hard to send for debugging.
We should keep file based logging.

Reply:
We should not do file based logging as it is not correctly handled in logrotate. I changed the rub command to limit the log size.

Reply:
Thanks @Cellebyte for feedback, can you please tell more about logrotate issue, I will look into it. But file based logging is helpful anyways. :)

Reply:
It is but it then needs to be toggled as if you ran in development mode it wants to write to a location which is not writable by a normal user.

Reply:
Yes, as logging is customizable, so file based logging can easily be disabled or log file location can be changed as per user need.

BTW, Currently sonic_exporter image is built with default user "root" and sonic_exporter runs as root in the container hence /var/log/ location is writable by the process, Also In developer_mode the same image runs, just with additional flag "-e DEVELOPER_MODE="true", In this case as well sonic_exporter process should be able to write to default /var/log/ location. Or may be I need to know more about your development setup.

Reply:
Thats only true for the container runtime docker.
If you run sonic-exporter as a user process. Which would be the case if you run it in crio or podman then you can't write to such locations.
What we could do is change the log output to a local location when the container is started in development mode.

Reply:
@Cellebyte , Yes, That can be a solution. 
Will resolve the conversation and close pull request on receving updates for above said solution. :)